modules/gsp-cluster/variables.tf

```diff
@@ -194,3 +194,13 @@ variable "cluster_zone_ids" {
   default     = []
   description = "List of DNS zone IDs associated with the cluster"
 }
+
+variable "dockerhub_username" {
+  default     = ""
+  description = "The username to use when pulling from DockerHub"
+}
+
+variable "dockerhub_password" {
+  default     = ""
```

Comment:
non-blocking: presumably if these vars are set to `""` then things will break ... so not sure it makes sense to have default values, but it's probably fine